src/import/importd.c

```diff
@@ -1399,6 +1422,7 @@ static int help(const char *program_path) {
                "Options:"
                "  -h --help                 Show this help\n"
                "     --version              Show package version\n"
+               " --xml-introspect=INTERFACE Write XML introspection data\n"
```

Comment:
this should say "bus" or "D-Bus" somewhere. Maybe change the explanation to "Write D-Bus introspection data"?

I mean, the fact that it is *dbus* introspection is much more interesting than that it is in XML

Reply:
hence, also change lhs to `--bus-introspect=IFACE`?